RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEESProducer_cfi.py

```diff
@@ -2,5 +2,8 @@
 
 phase2StripCPEESProducer = cms.ESProducer("Phase2StripCPEESProducer",
   ComponentType = cms.string('Phase2StripCPE'),
-  parameters    = cms.PSet()
+  parameters    = cms.PSet(
+    LorentzAngle_DB = cms.bool(False),			
+    TanLorentzAnglePerTesla = cms.double(0.07)
```

Comment:
ok thanks , but then I see that here the Strip CPE is defined in this PR , what is the status/plans for  Pixel CPE in PS ?